hlf/chaincode/general.md

```diff
@@ -0,0 +1,14 @@
+# General Information
+
+To provide a central point information and prevent duplicate annotations, this document shall contain general information on our models / techniques and architecture, where possible.
+
+##  <a id="Date&Time" />Dates & Times
+
+Unless stated otherwise, all of our timestamps are conform to ISO 8601, which declares a timestamp to match one of the following patterns:

```

Comment:
```suggestion
Unless stated otherwise, all of our timestamps are conform to ISO 8601 INSTANT, which declares a timestamp to match one of the following patterns:

```